### PR TITLE
Updated Russian localisation.

### PR DESCRIPTION
**Pull request by WolfDevil**

### CODE REVIEW

**Comment by MoNTE48** [comment]
LGTM. Can be merged.

File: MonitorControl/Support/ru.lproj/Localizable.strings [hunk 1]
```diff
@@ -2,10 +2,10 @@
 "Advanced" = "Дополнительные";
 
 /* Shown in the alert dialog */
-"Are you sure you want to enable a longer delay? Doing so may freeze your system and require a restart. Start at login will be disabled as a safety measure." = "Are you sure you want to enable a longer delay? Doing so may freeze your system and require a restart. Start at login will be disabled as a safety measure.";
+"Are you sure you want to enable a longer delay? Doing so may freeze your system and require a restart. Start at login will be disabled as a safety measure." = "Вы уверены что хотите включить увеличенную задержку? Это может привести к зависанию системы и потребуется перезагрузка. Запуск при старте системы будет выключен в мерах безопасности.";
 
 /* Shown in the alert dialog */
-"Are you sure you want to reset all preferences?" = "Are you sure you want to reset all preferences?";
+"Are you sure you want to reset all preferences?" = "Вы уверены что хотите сбросить все настройки?";
```
Comment (dm-zharov) [inline MonitorControl/Support/ru.lproj/Localizable.strings:8]:
```suggestion
"Are you sure you want to reset all preferences?" = "Вы уверены, что хотите сбросить все настройки?";
```

File: MonitorControl/Support/ru.lproj/Localizable.strings [hunk 2]
```diff
@@ -2,10 +2,10 @@
 "Advanced" = "Дополнительные";
 
 /* Shown in the alert dialog */
-"Are you sure you want to enable a longer delay? Doing so may freeze your system and require a restart. Start at login will be disabled as a safety measure." = "Are you sure you want to enable a longer delay? Doing so may freeze your system and require a restart. Start at login will be disabled as a safety measure.";
+"Are you sure you want to enable a longer delay? Doing so may freeze your system and require a restart. Start at login will be disabled as a safety measure." = "Вы уверены что хотите включить увеличенную задержку? Это может привести к зависанию системы и потребуется перезагрузка. Запуск при старте системы будет выключен в мерах безопасности.";
```
Comment (dm-zharov) [inline MonitorControl/Support/ru.lproj/Localizable.strings:5]:
```suggestion
"Are you sure you want to enable a longer delay? Doing so may freeze your system and require a restart. Start at login will be disabled as a safety measure." = "Вы уверены, что хотите включить увеличенную задержку? Это может привести к зависанию системы и потребовать перезагрузки. Запуск при старте системы будет выключен в качестве меры безопасности.";
```

Comment (MoNTE48) [reply MonitorControl/Support/ru.lproj/Localizable.strings:5]: Может, " и потребовать перезагрузку"?

Comment (dm-zharov) [reply MonitorControl/Support/ru.lproj/Localizable.strings:5]: Да 👌🏻

**Comment by waydabber** [comment]
Hey all, the new 3.0.0 version has changed most of the interface and localization strings (this is true for existing localizations as well), so we'll need to refresh the translation files. It would be best to start from the new strings as a baseline and do a pull request based on that.

Because of this, I'll close this PR for now, but please feel free to comment if you think an other approach would be more beneficial.

Thank you!

**Pull request closed without merging**